src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -101,8 +106,9 @@ public Map<CreationEnum, List<?>> addUsers(List<PiUser> users, String issuerId)
             } else if (constraintViolationSet.isEmpty()) {
                 PiUser addedUser = userRepository.save(user);
                 createdAccounts.add(addedUser.getUserId());
-
-                log.info(writeLog(issuerId, UserActions.CREATE_ACCOUNT, addedUser.getUserId().toString()));
+                if (issuerId != null) {
```

Comment:
Although this will only really be the case for testing, it feels wrong to have the logging as an if in the main code.

Could an issuer ID be passed through from the endpoint (potentially just TESTING_SUPPORT as pretty sure a string will work)

Reply:
Now pass in the issueID from the controller class :)